Traceback (most recent call last):
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mriki/.rye/py/cpython@3.11.8/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/mriki/workspace/prpy/.venv/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
linkedlist = SinglyLinkedList()
print(linkedlist)
linkedlist.remove(1)
------------------

----- stdout -----
...
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[5], line 3[0m
[1;32m      1[0m linkedlist [38;5;241m=[39m SinglyLinkedList()
[1;32m      2[0m [38;5;28mprint[39m(linkedlist)
[0;32m----> 3[0m [43mlinkedlist[49m[38;5;241;43m.[39;49m[43mremove[49m[43m([49m[38;5;241;43m1[39;49m[43m)[49m

Cell [0;32mIn[3], line 47[0m, in [0;36mSinglyLinkedList.remove[0;34m(self, data)[0m
[1;32m     43[0m     current [38;5;241m=[39m current[38;5;241m.[39mnext
[1;32m     45[0m [38;5;66;03m# リスト内に削除対象が存在しなかった場合[39;00m
[1;32m     46[0m [38;5;28;01melse[39;00m:
[0;32m---> 47[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124m"[39m[38;5;124mLinkedList.remove(data): data not in LinkedList[39m[38;5;124m"[39m)
[1;32m     48[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m

[0;31mValueError[0m: LinkedList.remove(data): data not in LinkedList

